pkg/orchestration/wiring/wiringutil/compute/secret_test.go

```diff
@@ -179,32 +178,14 @@ func TestGenerateEnvVarsEmptyVars(t *testing.T) {
 	assert.Len(t, envVars, 0)
 }
 
-func constructBindingResult(consumerName string, dependencyName string, smithServiceInstanceName string, prefix string, vars map[string]string) BindingResult {
+func constructBindingResult(consumerName string, dependencyName voyager.ResourceName, serviceInstance smith_v1.ResourceName, prefix string, vars map[string]string) BindingResult {
 	return BindingResult{
 		// This binding result describes a dependency to a resource in the State
-		ResourceName: voyager.ResourceName(dependencyName),
+		ResourceName: dependencyName,
 
 		// The shape is the shape output by that particular resource
 		// In this case, it's always a BindableEnvVar shape.
-		BindableEnvVarShape: &knownshapes.BindableEnvironmentVariables{
-			ShapeMeta: wiringplugin.ShapeMeta{
-				ShapeName: knownshapes.BindableEnvironmentVariablesShape,
-			},
-			Data: knownshapes.BindableEnvironmentVariablesData{
-				// This describes the instance that we should bind to
-				BindableShapeStruct: wiringplugin.BindableShapeStruct{
-					ServiceInstanceName: wiringplugin.ProtoReference{
-						Resource: smith_v1.ResourceName(smithServiceInstanceName),
-						Path:     "metadata.name",
-						Modifier: smith_v1.ReferenceModifierBindSecret,
```

Comment:
This was not right. `metadata.name` is not inside of a secret. I used the helper function and everything is correct now. Also changed the helper to accept prefix and vars to ensure we never forget them.